### PR TITLE
the last changes from trello tasks

### PR DESCRIPTION
**Pull request by boazmatz**
what I add in this update-

**general updates**
1.add favicon
2.add to Heroku columns product page working well
3.add more products

**Home page-**
1. add AppBar
2.space between products
3.new look - each product have photo and one button. buttons green color like product page
4.mobile view 3 products in each row 
5.Pressing on product button navigate the user into the product page

**product page-**
1. add AppBar
2.fix download button
3.mobile view 3 buttons are in the same row

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,6 +22,7 @@
     "express": "^4.17.1",
     "pg": "^7.18.2",
     "vue": "^2.6.11",
+    "vue-fragment": "^1.5.1",
```
Comment (roeeyud) [inline package.json:25]: Why is this here?

File: templates/homePage.html [hunk 2]
```diff
@@ -1,4 +1,25 @@
     <div class="thumbnail-container">
+      <nav class="navbar navbar-expand-lg navbar-light bg-light">
```
Comment (roeeyud) [inline templates/homePage.html:2]: Why is this page hard coded?aviehu didn't you sort this out? Where is vue-repeat

**Comment by boazmatz** [comment]
what I add in the last commit- **home and product page mobile view text looksBetter**

mobile text at product page and Home page looks really better in the same row

**Comment by roeeyud** [comment]
@boazmatz  make sure you run `npm install` and commit again to revert changes made to package.lock file
